### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrii256**
[DEMO LINK](https://andrii256.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/User/User.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './User.scss';
+
+export const User = ({ name }) => (
+  <p className="User">
+    directed by
```
Comment (rmnkk) [inline src/User/User.jsx:7]: You could use Template literals here

File: src/TodoList/TodoList.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+li::before {
+  content: "bruce li";
```
Comment (rmnkk) [inline src/TodoList/TodoList.scss:2]: https://imgur.com/dKNwvgy

**Review by rmnkk — APPROVED** [review]
Nice👍

File: src/Todo/Todo.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+export const Todo = ({ title, completed, user }) => (
+  <>
+    <div className={completed
+      ? 'todo todo--completed'
+      : 'todo todo--not-completed'}
+    >
+      <span className={(completed
+        ? 'todo__status todo__status--completed'
+        : 'todo__status todo__status--not-completed')}
```
Comment (vsm2k17) [inline src/Todo/Todo.jsx:14]: https://www.npmjs.com/package/classnames

File: src/Todo/Todo.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from '../User';
+import './Todo.scss';
+
+export const Todo = ({ title, completed, user }) => (
+  <>
```
Comment (vsm2k17) [inline src/Todo/Todo.jsx:7]: тут можна повертати div, React.Fragment можна видалити

**Review by vsm2k17 — APPROVED** [review]

**Pull request closed without merging**